data/xml/N19.xml

```diff
@@ -4820,7 +5010,7 @@
       <abstract>There have been many studies on neural headline generation models trained with a lot of (article, headline) pairs. However, there are few situations for putting such models into practical use in the real world since news articles typically already have corresponding headlines. In this paper, we describe a practical use case of neural headline generation in a news aggregator, where dozens of professional editors constantly select important news articles and manually create their headlines, which are much shorter than the original headlines. Specifically, we show how to deploy our model to an editing support tool and report the results of comparing the behavior of the editors before and after the release.</abstract>
       <url>N19-2010</url>
       <doi>10.18653/v1/N19-2010</doi>
-      <attachment type="presentation">N19-2010.Presentation.pdf</attachment>
+      <video href="https://vimeo.com/361705360" tag="video"/>
     </paper>
```

Comment:
Somehow this attachment was removed.

Reply:
Thanks for the heads up! I just noticed that and corrected it in the latest version. I have one more change to make and then it should be done. I'm not sure what happened with my update script, but it dropped a few things for some odd reason. I'll have this resolved momentarily.